ceph/ceph.py

```diff
@@ -1598,13 +1611,7 @@ def remove_ceph_object(self, ceph_object):
         if ceph_object.role == "osd":
             self.get_allocated_volumes()[0].status = NodeVolume.FREE
 
-    def open_firewall_port(self, port, protocol):
-        """
-        Opens firewall port on the node
-        Args:
-            port(str): port, can be range
-            protocol(str): protcol
-        """
+    def configure_firewall(self):
```

Comment:
It is a good practice to have a docString... as we use `readthedocs`... it helps describe the method.